bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/OptionsConstraint.java

```diff
@@ -55,7 +55,7 @@ default boolean isEnforceEnum() {
      * The length of enum and enumNames array must match
      *
      * @return the list of enum names
-     * @since com.adobe.cq.forms.core.components.models.form 0.0.1
+     * @since com.adobe.cq.forms.core.components.models.form 5.0.0
      */
-    String[] getEnumNames();
+    RichText[] getEnumNames();
```

Comment:
This can't be changed since it already exposed, mark it as deprecate and introduce a new function. Please mention in the new function, that this is introduced since which version based on package-info and only bump the major version